Expensify/App
# [User Settings] Allow users to modify avatar photo

### Details
- Refactored createmenu to accept menu items as props
- Added upload/delete avatar functionality
- Delete avatar will only populate if avatar is not a default avatar
- [existing bug] Popover on web/desktop appears in bottom left corner. https://github.com/Expensify/Expensify.cash/issues/2090

### Fixed Issues
Fixes https://github.com/Expensify/Expensify.cash/issues/2053

### Tests
1. Go to Settings -> Profile by clicking your avatar
2. Click Edit Photo, and upload a new photo
3. If you remove or have a default photo, the remove photo option should not appear

### QA Steps
same as above

### Tested On


### Video Walkthrough

https://user-images.githubusercontent.com/12075600/112771126-2ec3e080-8fdf-11eb-9311-431da950af24.mov

### Screenshots

#### Web
![Screen Shot 2021-03-28 at 3 46 25 PM](https://user-images.githubusercontent.com/12075600/112771136-44390a80-8fdf-11eb-812c-a9b0121741c5.png)

#### Mobile Web
![Screen Shot 2021-03-28 at 3 50 00 PM](https://user-images.githubusercontent.com/12075600/112771134-3c796600-8fdf-11eb-95ae-0d6fe66334b6.png)

#### Desktop
![Screen Shot 2021-03-28 at 3 48 16 PM](https://user-images.githubusercontent.com/12075600/112771140-49965500-8fdf-11eb-8fe0-08ae177dc3f8.png)

#### iOS
![Screen Shot 2021-03-28 at 3 46 58 PM](https://user-images.githubusercontent.com/12075600/112771155-5ca92500-8fdf-11eb-86e8-17b2d30837e3.png)
![Screen Shot 2021-03-28 at 3 47 16 PM](https://user-images.githubusercontent.com/12075600/112771157-5e72e880-8fdf-11eb-8b48-809a1f036a02.png)

#### Android
![Screen Shot 2021-03-28 at 3 52 09 PM](https://user-images.githubusercontent.com/12075600/112771160-63379c80-8fdf-11eb-825e-43fbdf564519.png)
